test/integration.js

```diff
@@ -46,7 +46,7 @@ function configureDefaultDC (dc) {
 // 4. test opening an already configured account (re-open above)
 
 test('setUp dc context', t => {
-  t.plan(13)
+  t.plan(18)
```

Comment:
This should probably be `17`. It seems the integration tests never finish on e.g. AppVeyor.

Reply:
Also doesn't finish on Travis https://travis-ci.org/deltachat/deltachat-node/jobs/552376198

Reply:
Hmm - strange in my local tests it worked. Will check again...